test/e2e/config/config.go

```diff
@@ -44,6 +44,8 @@ type Config struct {
 	ResourceGroup       string `envconfig:"RESOURCE_GROUP" default:""`
 	SoakClusterName     string `envconfig:"SOAK_CLUSTER_NAME" default:""`
 	ForceDeploy         bool   `envconfig:"FORCE_DEPLOY" default:"false"`
+	PublicSSHKey        string `envconfig:"PUBLIC_SSH_KEY" default:""`
+	PrivateSSHKeyPath   string `envconfig:"PRIVATE_SSH_KEY_FILE" default:""`
```

Comment:
It would be nice to update the field name or add a comment stating that this is a relative path